vcluster/configure/vcluster-yaml/sleep-mode.mdx

```diff
@@ -0,0 +1,196 @@
+| "sleepmode.loft.sh/ignore-ingresses" | ignore requests to associated ingresses  | "true" OR "false"  |
+| "sleepmode.loft.sh/ignore-groups" | ignore requests with a specific apigroup | "apps,group2,group3" |
+| "sleepmode.loft.sh/ignore-resources" | ignore requests to specific resource types  | "pods,resource2,resource3"  |
+| "sleepmode.loft.sh/ignore-verbs" | ignore requests with specific verbs | "create,verb1,verb2"  |
+| "sleepmode.loft.sh/ignore-resource-verbs" | ignore requests with specific verbs to specific resources |  "myresource.mygroup=create update delete, myresource2.mygroup=create update" |
+| "sleepmode.loft.sh/ignore-resource-names" | ignore requests to specific resources with specific names | "myresource.mygroup=name1 name2,myotherresource.mygroup=name1"  |
+| "sleepmode.loft.sh/ignore-user-agents" | ignore specific useragents with trailling wildcard support | "kubectl*,argo,useragent3"  |
+
+:::note
+These annotations are to be added to the vCluster workload, e.g. the `StatefulSet` or `Deployment` running the virtual cluster, and any request can be ignored dynamically by adding the header `X-Sleep-Mode-Ignore`.
+:::
+
+## Differences and compatibility with the platform
+
+The documentation on this page describes sleep mode for a vCluster without an agent, where workloads are scaled down, but the control plane remains active to monitor for activity that should wake it up.
+When a vCluster is connected to the platform via an agent, the agent can fully shut down the control plane, reducing resource usage further. Additionally, auto-deletion is only available when an agent is present.
+
+### Connecting to the platform
+
+When a vCluster, configured for sleep mode, is connected to the platform, it ceases trying to manage its own sleep mode actions, allowing the agent to take over.
+This requires compatible versions between vCluster and platform. Below is a table of compatibility and additional steps needed to be performed when installing or uninstalling an agent
+with sleep mode configured.
+<br/>
+
+| vCluster Version | Platform Version | Compatibility |  Installing an agent | Uninstalling an agent |
+| -- | -- | -- | -- | -- |
+| 0.22.x | 4.2.x | ✅ | Manually update vCluster config, moving it from `experimental` to `external.platform`. Translate any durations like `90m` into equivalent seconds e.g. `5400`.  Schema validation prevents configuring both at the same time. | Configuration needs to be manual reverted. |
+| 0.23.x | 4.2.x | ✅ | Manually update vCluster config, moving it from `experimental` to `external.platform`. Translate any durations like `90m` into equivalent seconds e.g. `5400`.  Schema validation prevents configuring both at the same time. | Configuration needs to be manual reverted. |
+| 0.24.0 | &lt;4.3.0 | ❌ | These versions are not compatible, as the vCluster version is ahead of the platform, causing the vcluster creation to be rejected.   | N/A |
```

Comment:
Following up from my question about config from about, in what cases would it be rejected ? Is it only when `external.platform` is configured ? 
It's not a blanket rejection for all cases right, that would be bad, it should be possible to  run 0.24 with 4.2 something. Though it makes sense that full sleep wouldn't work in this scenario, but I was assuming that was still possible to configure workload only sleep.

Reply:
It is a blanket rejection because the `0.24.xxx` image has the new schema that the older platform doesn't know about.  So from the UI's point of view the config will be `experimental...`, and will allow you to save with that configuration.  It supplies those values to a `0.24.xxx` image that fails validating against the new schema.

Reply:
Ahh ok, I suppose that makes sense. So that would happen regularly when config changes in any situation like this.